tasks-by-users/Naufil-Shemle/02-html-code-with-harry/html-tut7.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tables and Lists</title>
+  </head>
+
+  <body>
+    <!-- ul types disc, square and circle-->
+    <ul type="circle">
+      <li>This is First item of my unordered list</li>
+      <li>This is Second item of my unordered list</li>
+      <ul>
+        <li>Another One</li>
+        <li>Another Two</li>
+        <li>Another Three</li>
+      </ul>
+      <li>This is Third item of my unordered list</li>
+    </ul>
+    <ol type="A">
```

Comment:
what does this type do ?

Reply:
By this "type" attribute we can customize the list and when we type "type" in VScode it gives some recommendation for ordered list

Reply:
my question was.... with this `type="A"` what type of list would that be ? @Naufil-Shemle